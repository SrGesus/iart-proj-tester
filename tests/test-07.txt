FC	BB	BC	FB
FC	BD	FD	FD
VD	BC	BD	BD
FB	FB	VE	FD