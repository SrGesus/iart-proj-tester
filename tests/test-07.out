FD	BB	BB	FE
FD	BE	FC	FB
VB	BC	BB	BE
FC	FD	VC	FC
